lib/Item.dart

```diff
@@ -0,0 +1,51 @@
+import 'package:dart_application_1/constants.dart';
+import 'InputOutputItemsUtility.dart';
+
+class Item {
+  String itemName;
+  ItemTypes itemType;
+  double itemPrice;
+  double itemQuantity;
```

Comment:
If the value of these members are not changing once initialised, then make them final.